### PR TITLE
Fix CI creating file

### LINKED ISSUE #33: Fix running unit tests on the CI
# Summary
Some of the features for the database and the REST API require accessing files on disk, which we do not have access to on Github. There are probably some way to get around this or mock accessing the disks. However, it is not  a very high priority right now provided that developers run the test suite on their own first. 

# Example
An example of the way that it is currently broken is this run of the CI during the development of the database: https://github.com/baas-project/baas/runs/4091566066?check_suite_focus=true

### PR DESCRIPTION
**Pull request by u5surf**
Fixes #33

### CODE REVIEW

**Comment by ValentijnvdBeek** [comment]
HI @u5surf 

It looks great! There are two small issues, but if that is resolved we can merge it could. Is it possible to create and add yourself to the AUTHORS file? Also, please object if you have any issue with licensing your code under a free license later on. 

Thanks a lot!

Valentijn

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,17 +6,26 @@ jobs:
 # whenever you do end up solving this issue so that I know how many years have passed.
 # Until that time, please manually run the test suite before pushing. Thanks. 
 #   - ValentijnvdBeek
```
Comment (ValentijnvdBeek) [inline .github/workflows/ci.yml]: The comment here is obsolete so it would be nice if you where to delete it.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,17 +6,26 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Check version
+        run: |
```
Comment (ValentijnvdBeek) [inline .github/workflows/ci.yml]: I understand that this is here for debugging reasons. However would it not be better to remove this step before merging it? Or is there something I am missing?

**Review by ValentijnvdBeek — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (u5surf) [reply .github/workflows/ci.yml]: @ValentijnvdBeek I've just removed it.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (u5surf) [reply .github/workflows/ci.yml]: @ValentijnvdBeek 
Ah, It's my fault. That is just using for debugging, Then, I removed it.

**Comment by ValentijnvdBeek** [comment]
Thanks looks great!

**Pull request merged**